packages/react-core/src/demos/DataListDemo.md

```diff
@@ -123,9 +124,11 @@ class ExpandableDataList extends React.Component {
                     variant="plain"
                     aria-label={this.state.allExpanded ? 'Collapse all rows' : 'Expand all rows'}
                   >
-                    <ToolbarExpandIconWrapper>
-                      <AngleRightIcon />
-                    </ToolbarExpandIconWrapper>
+                    <Icon shouldMirrorRTL>
+                      <ToolbarExpandIconWrapper>
+                        <AngleRightIcon />
+                      </ToolbarExpandIconWrapper>
+                    </Icon>
```

Comment:
Oh looks like this is something we missed in core. I put up a PR to fix it here - https://github.com/patternfly/patternfly/pull/5937

Reply:
Ah yeah I was considering putting a core PR up as well, but since we're not hardcoding the icon i ended up going with the example update instead. If we'd want to go the CSS route we may want to consider hardcoding this icon rather than allowing a consumer to pass any icon they want.

Reply:
Ah! I wasn't keeping in mind that the icon is passed explicitly. I think we should definitely provide the icon the same way we do with the data-list and table, which is the primary use case for this toggle in the toolbar 

https://github.com/patternfly/patternfly-react/blob/dae070ac137be3eb7ac68a6ca14b391dc6fc799b/packages/react-core/src/components/DataList/DataListToggle.tsx#L48

https://github.com/patternfly/patternfly-react/blob/dae070ac137be3eb7ac68a6ca14b391dc6fc799b/packages/react-table/src/components/Table/utils/decorators/treeRow.tsx#L58

Though I'm assuming that's a breaking change? If so I'll back the change out in core, use the icon component like you have here, and we can create a breaking change issue to update that later?

Reply:
@tlabaj wdyt? Would make sense to use the same icon used most other places for expansion here.

Reply:
Yes, rendering the icon internally would be breaking since they can technically pass in whatever icon they want right now.